transmogrifier/sources/marc.py

```diff
@@ -283,6 +430,65 @@ def get_optional_fields(self, xml: Tag) -> dict:
         # publication_information
 
         # related_items
+        related_item_marc_fields = [
+            {
+                "tag": "765",
+                "subfields": "abcdghikmnorstuwxyz",
+                "relationship": "Original Language Version",
+            },
+            {
+                "tag": "770",
+                "subfields": "abcdghikmnorstuwxyz",
+                "relationship": "Has Supplement",
+            },
+            {
+                "tag": "772",
+                "subfields": "abcdghikmnorstuwxyz",
+                "relationship": "Supplement To",
+            },
+            {
+                "tag": "780",
+                "subfields": "abcdghikmnorstuwxyz",
+                "relationship": "Previous Title",
+            },
+            {
+                "tag": "785",
+                "subfields": "abcdghikmnorstuwxyz",
+                "relationship": "Subsequent Title",
+            },
+            {
+                "tag": "787",
+                "subfields": "abcdghikmnorstuwxyz",
+                "relationship": "Related Item",
```

Comment:
We should probably default to `relationship=Not specified` here like we do for our other sources, for consistency.